README.md

```diff
@@ -40,9 +40,9 @@ Otherwise, you can install the [`autocomplete-core`](https://www.algolia.com/doc
 All Autocomplete packages are available on the [npm](https://www.npmjs.com) registry.
 
 ```bash
-yarn add @algolia/autocomplete-js
+yarn add algoliasearch search-insights @algolia/autocomplete-js
 # or
-npm install @algolia/autocomplete-js
+npm install algoliasearch search-insights @algolia/autocomplete-js
```

Comment:
I'm not positive we should ask users to install algoliasearch and search-insights just to fix peer dependencies warnings, as they're not needed with third-party sources.

Maybe instead we could add a small "Troubleshooting" "FAQ" section in the Readme, to explain why we have those peer dependencies, something like:

> Q: Why are there multiple peer dependencies warnings when I install autocomplete-js?
> A: Autocomplete provides first-class support for Algolia sources and sending events. This means we rely on some packages that you might not need by default. These warnings can be safely ignored.